### ISSUE TITLE
JSPM issue

### ISSUE THREAD

**Issue by Bidyuta** [jspm/jspm-cli#2217]
warn Error on download for github:systemjs/systemjs
     Error: connect ETIMEDOUT 192.30.253.116:443
         at Object.exports._errnoException (util.js:1022:11)
         at exports._exceptionWithHostPort (util.js:1045:20)
         at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1087:14)

err

err  Error downloading loader files.

**Comment by Bidyuta** [comment]
when I will run the command like jspm init after that I got error. Please help me.

**Comment by fusionstrings** [comment]
Please provide your operating system, Node, NPM version.

**Comment by Bidyuta** [comment]
Hi Team,

Thanks for response. Please find the details below :-

OS – Windows Server 2008 R2 Standard
Node – Node.js 6.9.4
NPM – 3.10.10


Regards,
Bidyuta Panigrahi

**Comment by lfrancke** [comment]
I see the same. It's been a while since I've used JSPM but this used to work in the past. Now it takes multiple (20+) tries to get a `jspm install` to work successfully.

I'm on MacOS 10.11.6 (El Capitan) with NPM 4.2.0 and JSPM 0.16.52.

This happens for me on the latest unchanged Aurelia Skeleton app.

I have configured the Github repo settings and the token does have the correct permission.

Anything I can do to help debug this I'm happy to do.
